src/script_lua.c

```diff
@@ -1383,13 +1383,23 @@ void luaSetTableProtectionRecursively(lua_State *lua) {
 }
 
 void luaRegisterVersion(lua_State* lua) {
+    /* For legacy compatibility reasons include Redis versions. */
     lua_pushstring(lua,"REDIS_VERSION_NUM");
     lua_pushnumber(lua,REDIS_VERSION_NUM);
     lua_settable(lua,-3);
 
     lua_pushstring(lua,"REDIS_VERSION");
     lua_pushstring(lua,REDIS_VERSION);
     lua_settable(lua,-3);
+
+    /* Now push the new table values*/
+    lua_pushstring(lua,"SERVER_VERSION_NUM");
+    lua_pushnumber(lua,SERVER_VERSION_NUM);
+    lua_settable(lua,-3);
+
+    lua_pushstring(lua,"SERVER_VERSION");
+    lua_pushstring(lua,SERVER_VERSION);
+    lua_settable(lua,-3);
```

Comment:
Shall we add "SERVER_NAME" too? 

(Note to self) This is how it's accessed from Lua:

    127.0.0.1:6379> eval 'return redis.REDIS_VERSION' 0
    "255.255.255"

Reply:
No strong opinion from me, if you think someone will use it.

Reply:
I don't know. Do you think anyone will user SERVER_VERSION? :)

I think it's good to add both together, just like we add both in INFO.